storage-operator/pkg/salt/client.go

```diff
@@ -42,6 +42,36 @@ func NewClient(creds *Credential) *Client {
 	}
```

Comment:
Commit message:

```diff
- Note that the name subject to change (because the Salt state is not
+ Note that the name is subject to change (because the Salt state is not
```

Reply:
Done